### DISCUSSION TITLE
does not support High DPI and multibyte(MFC APP)？

### DISCUSSION THREAD

**Discussion by gzhwl** [jovibor/HexCtrl#38] [Q&A] [answered]
Hi, Big Boss:
   I found this perfect Lib, I like to test it, I found it does not support High DPI and multibyte(MFC APP) now, do you have plan to perfect it？[](url)

**Comment by jovibor** [comment]
Hi.
MFC is supported only with **Unicode** charset, multibyte is very obsolete and should not be used for any fresh projects. I have no plans to support it.
As for High DPI, what is exact issue with it? Could you elaborate more?

**Comment by datasynergyuk** [comment]
Multibyte is ancient and modern projects (since the death of Windows 9x at least) shouldn't need it. For what it is worth, it would be huge job to change HexCtrl to work with multibyte. I cannot see it would be practical. That said, if a legacy project is multibyte it would equally be difficult to change that project. In this case, HexCtrl probably isn't suitable. Maybe someone else would like to write a multibyte wrapper for HexCtrl. That would be fun...

**Comment by jovibor** [comment, marked as answer]
As of [HexCtrl 3.8 release](https://github.com/jovibor/HexCtrl/releases/tag/v3.8), full [Per-Monitor V2 DPI Awareness](https://learn.microsoft.com/en-us/windows/win32/hidpi/high-dpi-desktop-application-development-on-windows#per-monitor-and-per-monitor-v2-dpi-awareness) support is implemented.
It's the most modern High-DPI mode in Windows at the moment.
